modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -385,4 +385,25 @@ definition {
 			actionName = "Update Action Name");
 	}
 
+	@description = "LPS-146871 - Verify that the Action name is required"
+	@priority = "4"
+	test VerifyActionNameIsRequired {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 1",
+			objectName = "CustomObject1",
+			pluralLabelName = "Custom Objects 1");
```

Comment:
It looks like the file is using an older naming convention. For now, I'm going to enforce the using the ticket number until we decide on a different approach.